### DISCUSSION TITLE
Save entered authorization token in localstorage

### DISCUSSION THREAD

**Discussion by max13fr** [scalar/scalar#1572] [Ideas]
Hello,

It would be awesome if we can have an option that allow to save Authentification parameter (like the token) in the local storage, to reuse it if the use refresh or close & reopen later the documentation.

Thanks in advance,
Best regards, Max

**Comment by filippobellei** [comment]
Another useful thing would be to pre-fill the Authorization header with the Bearer keyword

**Reply by hanspagel** [reply]
Wait… this should be done already. Can you share the relevent part of your OpenAPI specification?

**Reply by filippobellei** [reply]
```json
{
    "components": {
        "schemas": {
            "securitySchemes": {
                "Bearer": {
                    "type": "apiKey",
                    "name": "Authorization",
                    "in": "header"
                }
            }
        }
    },
    "security": [
        {
            "Bearer": []
        }
    ]
}
```

**Comment by hanspagel** [comment]
I’ve created an issue here: https://github.com/scalar/scalar/issues/2005

**Comment by amritk** [comment]
This now works!

https://github.com/scalar/scalar/blob/main/documentation/configuration.md#persistauth-boolean
